website/docs/taskfile_versions.mdx

```diff
@@ -60,6 +60,13 @@ Now when someone tries to run the Taskfile with an older version of Task, they
 will receive an error prompting them to upgrade their version of Task to
 `v3.17.0` or greater.
 
+:::info
+
+Note that this functionality was introduced in [v3.34.0](https://github.com/go-task/task/releases/tag/v3.34.0),
+so older versions are not able to give you this warning.
+
+:::
+
```

Comment:
Shorter hint?

> Semantic version checking was introduced with v3.34.0 of Task.


And updating the examples earlier in the text to use versions _after_ v3.34.0